.lintr

```diff
@@ -0,0 +1 @@
+linters: with_defaults(line_length_linter(120))
```

Comment:
linting essentially means styling. linters are things you can configure for your code projects to help you say things like "make sure people always name their variables like_this and not likeThis" or "make sure we dont have any variables being created that are never even used". This will help maintain the integrity of your code and can be checked automatically every time you submit new work